src/ray/core_worker/core_worker.h

```diff
@@ -126,21 +126,21 @@ class TaskCounter {
   CounterMap<std::pair<std::string, bool>> running_in_get_counter_ ABSL_GUARDED_BY(&mu_);
   CounterMap<std::pair<std::string, bool>> running_in_wait_counter_ ABSL_GUARDED_BY(&mu_);
 
-  std::string job_id_ ABSL_GUARDED_BY(&mu_) = "";
+  std::string job_id_ ABSL_GUARDED_BY(&mu_);
   // Used for actor state tracking.
-  std::string actor_name_ ABSL_GUARDED_BY(&mu_) = "";
+  std::string actor_name_ ABSL_GUARDED_BY(&mu_);
   int64_t num_tasks_running_ ABSL_GUARDED_BY(&mu_) = 0;
 };
 
 struct TaskToRetry {
   /// Time when the task should be retried.
-  int64_t execution_time_ms;
+  int64_t execution_time_ms{};
```

Comment:
Just a question, curious why not `= 0`? I know default ctor has the same effect

Reply:
this was the autofix, not me, i'd prefer = 0 lol